0.3.0/cli/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Build OpenRCT2
+FROM ubuntu:20.04 AS build-env
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
+ && apt-get install --no-install-recommends -y git cmake pkg-config ninja-build clang-10 duktape-dev libsdl2-dev libspeexdsp-dev libjansson-dev libcurl4-openssl-dev libcrypto++-dev libfontconfig1-dev libfreetype6-dev libpng-dev libzip-dev libssl-dev libicu-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && ln -s /usr/bin/clang-10 /usr/bin/clang \
+ && ln -s /usr/bin/clang++-10 /usr/bin/clang++
+
+ENV OPENRCT2_REF v0.3.0
+WORKDIR /openrct2
+RUN git -c http.sslVerify=false clone --depth 1 -b $OPENRCT2_REF https://github.com/OpenRCT2/OpenRCT2 . \
+ && mkdir build \
+ && cd build \
+ && cmake .. -G Ninja -DCMAKE_BUILD_TYPE=release -DCMAKE_INSTALL_PREFIX=/openrct2-install/usr -DCMAKE_INSTALL_LIBDIR=/openrct2-install/usr/lib \
```

Comment:
This image is intended for end users who want to just run the server. `-DCMAKE_INSTALL_LIBDIR=/openrct2-install/usr/lib` makes me think there's a static library installed in there as well, can you also check that, please?

Reply:
Looks like the only thing in `/openrct2-install/usr/lib/` is `libopenrct2.a`

Reply:
This would be considered a `-dev` part of a pa page, not useful at all to run the server. Can you remove it?

Reply:
That saves another 35MB! Let me know if you would've done it differently.